src/components/shared-styles.js

```diff
@@ -33,28 +33,28 @@ export const SharedStyles = html`
   }
 
   h2 {
-    font-size: 24px;
+    font-size: 1.5rem;
     text-align: center;
     color: var(--app-dark-text-color);
   }
 
-  @media (min-width: 460px) {
+  @media (min-width: 28.75em) {
     h2 {
-      font-size: 36px;
+      font-size: 2.25rem;
     }
   }
 
   .circle {
     display: block;
-    width: 64px;
-    height: 64px;
+    width: 4em;
+    height: 4em;
     margin: 0 auto;
     text-align: center;
     border-radius: 50%;
     background: var(--app-primary-color);
     color: var(--app-light-text-color);
-    font-size: 30px;
-    line-height: 64px;
+    font-size: 1.875rem;
+    line-height: 2.133333333;
```

Comment:
Realizing only now it's a circle/art-direction use case; it's fine changing the line-height to also be 4rem here.